base/reduce.jl

```diff
@@ -396,7 +396,11 @@ end
 
 
 ## prod
+"""
+    prod(f, iter)
```

Comment:
WOW I AM SUCH AN IDIOT. I have `::Callable` in the docstring in another local commit that I didn't push that isn't present here. I'm so sorry for wasting your time with this!